.circleci/config.yml

```diff
@@ -117,8 +117,8 @@ commands:
       - run:
           name: Migrate and seed database
           command: |
-            cf run-task << parameters.app_name >> "yarn db:migrate:prod"
-            cf run-task << parameters.app_name >> "yarn db:seed:prod"
+            cf run-task << parameters.app_name >> \
+            "yarn db:migrate:prod && yarn db:seed:prod"
```

Comment:
I think we might still run into a race... We might have to do `yarn db:seed:undo && yarn db:migrate:undo &&  yarn db:migrate && yarn db:seed` in one task. Otherwise we could start migrating the db before the `undo` task has completed (or started).

Reply:
I was thinking this wouldn't create a race condition because of circle ci job step order and db locking. In the steps, the undo seed/migrate task is submitted to cf and then cf has to respond that it successfully submitted the task. After circle ci gets this success message back, it starts the next step of the job, submitting the seed/migrate task. By then the db should be locked right? I know sequelize (and node in general) are built on async, but running migrations should enforce some order right? You can't run migrate and undo migrate at the same time, right?

Reply:
I'm not sure what locking goes on, but regardless it seems there is at least chance between `seed:undo` and `migrate:undo` for the `db:migrate` (from the second step) to fire off. Not only that but these jobs are just submitted to cf so it seems possible for them to run out of order (even though the "undo" job hits the queue first) depending on how the first job is "queued".

Reply:
Yeah you're probably right. (It was just more elegant the other way- chaining commands seems so hacky.) But yeah, that'll ensure proper execution order and prevent devs from having to go check cf application logs- which is what's actually important.

Reply:
> (It was just more elegant the other way- chaining commands seems so hacky.)

Yeah I definitely agree!